index.html

```diff
@@ -0,0 +1,230 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Zombie</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+</head>
+<body>
+    <header class="main-header">
+        <a class="main-header__link top-header__link--selected" href="https://www.figma.com/file/sqBxKrXay0SKioyK20svju/ZombieUI?node-id=117%3A582">ZombieUI</a>
+        <span class="main-header__span">Hello, Guest</span>
+    </header>
+    <main class="main-content">
+        <h1 class="main-content__h1">Home page</h1>
+        <div class="big">
+            <div class="group">
+                <header class="group__header">
+                    Imp
+                </header>
+                <main class="group__main">
+                    <div class="group__main__header">
+                        Takes double damage
+                    </div>
+                    <div class="group__main__content">
+                        <div class="content__property">
+                            <div class="row">
+                                <div class="cell cell-name">
+                                    Health:
+                                </div>
+                                <div class="cell cell-value">
+                                    10
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="cell cell-name">
+                                    Speed:
+                                </div>
+                                <div class="cell cell-value">
+                                    4
+                                </div>
+                            </div>
+                            <div class="row">
+                                <div class="cell cell-name">
+                                    Damage:
+                                </div>
+                                <div class="cell cell-value">
+                                    1
+                                </div>
+                            </div>
+                        </div>
+                        <div class="content__img">
+                            <img class="img__zombie" src="pic/01-imp.png" alt="zombie"/>
```

Comment:
It does not match figma mockups
![image](https://user-images.githubusercontent.com/48202430/160796885-3afa0c94-e5a4-491c-bd3a-1a1fbe90d06b.png)

Reply:
fixed